k8s/arroyo/templates/NOTES.txt

```diff
@@ -6,11 +6,11 @@ Once the release is fully deployed, you can navigate to the web ui by running
 
   $ open "http://$(kubectl get service/{{ .Release.Name }}-api -o jsonpath='{.spec.clusterIP}')"
 
-(note this might take a minute or so if you are also deploying Postgres)
+(note this might take a few minutes so if you are also deploying Postgres)
```

Comment:
wording. should be "note that this might take a few minutes if you are also deploying Postgres"